docs/changes.rst

```diff
@@ -11,7 +11,8 @@ skops Changelog
 
 v0.10
 ----
-
+- Removes Pythn 3.8 support and adds Python 3.12 Support :pr:`418` by :user:`Thomas Lazarus <lazarust>`.
+- Removes `sklearn-intelex` as a dependency :pr:`420` by :user:`Thomas Lazarus <lazarust>`.
```

Comment:
this was never a dependency of the package. We can write:

```suggestion
- Removes a shortcut to add `sklearn-intelex` as a not dependency.
  :pr:`420` by :user:`Thomas Lazarus < lazarust > `.
```

Reply:
Ah, so it wasn't technically a dependency since we were dynamically using it if it existed? Do I have the understanding of that right?

Reply:
Yes, correct. It was only a test/doc dependency. Not an install dependency